src/udiskslinuxencrypted.c

```diff
@@ -1194,11 +1194,148 @@ handle_resize (UDisksEncrypted       *encrypted,
 
 /* ---------------------------------------------------------------------------------------------------- */
 
+/* runs in thread dedicated to handling method call */
+static gboolean
+handle_convert (UDisksEncrypted       *encrypted,
+                GDBusMethodInvocation *invocation,
+                const gchar           *target_version,
+                GVariant              *options)
+{
+  UDisksObject *object = NULL;
+  UDisksBlock *block;
+  UDisksDaemon *daemon;
+  UDisksState *state = NULL;
+  uid_t caller_uid;
+  const gchar *action_id = NULL;
+  const gchar *message = NULL;
+  GError *error = NULL;
+  UDisksBaseJob *job = NULL;
+  BDCryptoLUKSVersion bd_target_version;
+
+  object = udisks_daemon_util_dup_object (encrypted, &error);
+  if (object == NULL)
+  {
```

Comment:
Please note that the opening parenthesis should be indented by two spaces (and the inside block the same way).